src/tests/Parser/DisciplinePriest/Features/AtonementAttribution.test.js

```diff
@@ -0,0 +1,30 @@
+import AtonementAttribution from 'Parser/DisciplinePriest/Modules/Features/AtonementAttribution';
+import SPELLS from 'common/SPELLS';
+import processEvents from '../Fixtures/processEvents';
+
+import {
+  TwoSuccesiveDamagingEventsWithAtonementOnSelfBetween,
+  TwoSuccesiveDamagingEvents
+} from '../Fixtures/ReorderingScenarios';
+
+describe('DisciplinePriest.Reordering', () => {
+
+  let atonementAtribution;
+  beforeEach(() => {
+    atonementAtribution = new AtonementAttribution({
+      reorder: () => true,
+      toPlayer: () => true,
+      byPlayer: () => true,
+      toPlayerPet: () => false,
+      byPlayerPet: () => false,
+    });
+  });
+
+  it('Two Succesive Damaging Events with atonement on self between is reordered', () => {
+    let result = atonementAtribution.reorderEvents(TwoSuccesiveDamagingEventsWithAtonementOnSelfBetween);
```

Comment:
I'd much rather see the actual array here so that I can assess what the test does without having to jump between files.

Reply:
Agreed